src/commands/check_updates.rs

```diff
@@ -0,0 +1,34 @@
+use crate::check_update;
+
+use super::*;
+use serde_json::json;
+
+/// Test the update check
+#[derive(Parser)]
+pub struct Args {}
+
+pub async fn command(_args: Args, json: bool) -> Result<()> {
```

Comment:
We should output if we are on the latest version. Right now it doesn't output anything and is a bit confusing

![image](https://github.com/user-attachments/assets/2a1406bd-cae8-4d68-bd17-1aaefc7be01d)